src/main/java/io/vertx/mqtt/MqttClientOptions.java

```diff
@@ -382,6 +382,15 @@ public MqttClientOptions setMaxMessageSize(int maxMessageSize) {
     return this;
   }
 
+  /**
+   * Do the same thing as {@link MqttClientOptions#setKeepAliveTimeSeconds(int)}. Use it instead.
+   */
+  @Deprecated
+  @Override
+  public NetClientOptions setIdleTimeout(int idleTimeout) {
```

Comment:
the return type has to be `MqttClientOptions`

Reply:
Sure